force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,7 +29,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        String returnValue = null;
+        if (x > y)
+            returnValue = 'Hello World!';
+
+        return returnValue; // Replace null with the variable you used to store the result
```

Comment:
This is alright, but this is how I would structure methods like this.
- Generally in Apex (and programming in general) you return invalid values, errors, or throw exceptions early in the method. This is called "return early".
- when there's no need for a local variable and the method is short, you don't need to declare a variable like "returnValue". if you were to use a value like this, I would name it what it is rather than something generic like "return value". So, you would probably call this "result", "value" or best, "message".
- it is strongly preferred to always use curly braces when you have an if statement. you don't need them, but I would strongly suggest you use them 9/10 times.

```suggestion
        if (x == null || y == null || x <= y) {
            return null;
        }
        return 'Hello World!';
```